src/Components/VoteOnArticle.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { patchVotesOnArticle } from "../api";
+
+const VoteOnArticle = ({ votes }) => {
+  const [articleVotes, setArticleVotes] = useState(0);
+  const { article_id } = useParams();
+
+  const handleVotesonArticle = (increment) => {
+    setArticleVotes((currVotes) => currVotes + increment);
+    patchVotesOnArticle(article_id, { inc_votes: increment }).catch((error) =>
+      console.log("This page is not working now.Try later!")
```

Comment:
this `console.log` isn't good enough error handling!

Someone on who isn't a developer might not see it!

Reply:
Got it!, I will review it! :)